Polygon.js

```diff
@@ -0,0 +1,129 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?://(?:www\\.)polygon\\.com",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-12-06 02:04:51"
+}
+
+// http://en.wikipedia.org/wiki/User_talk:Czar
```

Comment:
Is this necessary? I saw that many translators did not have such a block. If it is, can I use a freer license than the GPL3 for the translator?